olm/prepare-community-operators-update.sh

```diff
@@ -49,7 +51,7 @@ Options:
 "
 }
 
-for platform in 'kubernetes' 'openshift'
+for platform in $PLATFORMS
```

Comment:
if a comma-separated list, would need to simply transform with `tr`:

```
$➔ for platform in abc,def; do echo $platform; done
abc,def
```

```
$➔ for platform in $(echo "abc,def"|tr "," " "); do echo $platform; done
abc
def
```
Thus:

```
for platform in $(echo $PLATFORMS | tr "," " ")
```